product_index.csv

```diff
@@ -0,0 +1,22 @@
+bgc_plankton,bgc_data/bgc_trip.dataresource.yaml
```

Comment:
This file is just an index used in the GitHub Actions tests to group the schemas by product, so we can easily run the tests independently for each product/group.